appveyor.yml

```diff
@@ -35,12 +40,16 @@ install:
   - if [%COMPILER%]==[clang-cl] conda config --add channels conda-forge --force
   - if [%COMPILER%]==[clang-cl] conda install --yes clangdev ninja cmake
   - if [%COMPILER%]==[clang-cl] call "C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\vcvarsall.bat" amd64
+  - pip install -q git+https://github.com/xoviat/clcache.git@findCompiler
```

Comment:
Don't merge this PR until the fix in merged to master, as I will probably delete the branch.